packer/build_image.sh

```diff
@@ -238,6 +245,7 @@ if [ "$BUILD_MODE" = "debug" ]; then
 fi
 if $DEBUG ; then
   IMAGE_NAME="debug-$IMAGE_NAME"
+  ENV_TAG="debug"
```

Comment:
Why to add it twice? in line 17 and here..

Reply:
leftover , removed